CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## vNext
+## 5.1.1
```

Comment:
I think we should add `vNext` in this PR. 
e.g.:
```suggestion
# Changelog

## vNext

## 5.1.1

### Chores
...
```

Reply:
Hi @tomislavhoracek thank you for the remark, but we do not have yet a vNext because there is no new PR, but as soon someone creates a new PR we include it :)